static/app/views/issueList/overview.tsx

```diff
@@ -333,6 +336,8 @@ class IssueListOverview extends Component<Props, State> {
   }
 
   getBetterPriorityParams(): BetterPriorityEndpointParams {
+    console.log(this.props.experimentAssignment);
+    console.log(this.props.organization?.experiments?.PrioritySortExperiment);
```

Comment:
@ceorourke there is a `useExperiment` hook you can use

Reply:
I used the `withExperiment` one that's in the docs